test/evergreen/asan_leaks.supp

```diff
@@ -6,46 +6,46 @@ leak:PyObject_Malloc
 
 # FIXME-WT-16083: Resolve the warnings and remove the suppressions.
```

Comment:
Should we close this one now since we have separate tickets for all the issues? Or can we move it to the [???](https://github.com/wiredtiger/wiredtiger/pull/12931/changes#diff-df3066dbeececcdc3b54ea669876cc57bb970ea77ad6fe1907dd653f081666e7R51) place